### ISSUE TITLE
`remdsort` uses `trajectory_output.format`, but with new ase format, extension is actually .extxyz and not .ase

### ISSUE THREAD

**Issue by svandenhaute** [i-pi/i-pi#361]
Title says it all, or at least how I understand it. I'm applying `remdsort` after a REX simulation. My trajectory output format is `ase`, which in recent i-PI versions means write an XYZ file but use the `.extxyz` extension. `TrajectoryOutput` has some logic **[here](https://github.com/i-pi/i-pi/blob/dee5fa1bb823e0198a51cdeaa4d7c49c52e73853/ipi/engine/outputs.py#L364) which changes the actual format from `ase` to `extxyz`, but it looks like `remdsort` and potentially other parts of i-PI assume that `format` actually is identical to the extension.

Bottom line: `remdsort` looks for a file `foo.ase` while it should actually be looking for `foo.extxyz`.

**Comment by mahrossi** [comment]
Yes! Indeed! Is it possible that you can implement an easy fix and open a pull request? :-)

**Comment by ceriottm** [comment]
Fixed by #365
